index.js

```diff
@@ -77,34 +72,20 @@ exports.clear = function() {
 };
 
 exports.get = function(key) {
+  var value = null;
+
   var data = cache[key];
-  if (typeof data != "undefined") {
-    if (isNaN(data.expire) || data.expire >= now()) {
-      if (debug) hitCount++;
-      return data.value;
-    } else {
-      // free some space
```

Comment:
what if you try to get() something that is expired, but the callback hasn't fired yet (because you're main loop is blocking the JS thread or something).